.ci/jenkins/prepare_environment.sh

```diff
@@ -60,9 +55,32 @@ function build_ngraph() {
     return 0
 }
 
-# Link Onnx models
-mkdir -p /home/onnx_models/.onnx
-ln -s /home/onnx_models/.onnx /root/.onnx
+function main() {
+    # By default copy stored nGraph master and use it to build PR branch
+    BACKEND="cpu"
+
+    PATTERN='[-a-zA-Z0-9_]*='
+    for i in "$@"
+    do
+        case $i in
+            --backend=*)
+                BACKEND="${i//${PATTERN}/}"
+                ;;
+            *)
+                echo "Parameter $i not recognized."
+                exit 1
+                ;;
+        esac
+    done
+
+    BUILD_CALL="build_ngraph \"/root\" \"${BACKEND}\""
+    # Link Onnx models
```

Comment:
ONNX should capitalized, it's an abbreviation.

Reply:
Done